axes/backends.py

```diff
@@ -7,6 +7,13 @@
 
 class AxesModelBackend(ModelBackend):
 
+    class RequestParameterRequired(Exception):
```

Comment:
This exception should really live in `axes.exceptions` as per good naming conventions. You just create that `exceptions.py` file. There is no need to put this inside the `AxesModelBackend` class, nesting classes just makes this harder to read and find.